aptstract/engines/postgres.py

```diff
@@ -1,10 +1,11 @@
 from pathlib import Path
 
-from ..paths import PG_RESTORE_PATH
+from ..paths import get_utility_path
 from ..streamrun import subprocess_stream
 
 
 def restore(config, file: Path):
+    PG_RESTORE_PATH = get_utility_path("pg_restore")
```

Comment:
usually all caps is reserved for constants in a module level, right? (this variable seems at function scope)

i'd just use lower case here unless you plan on keeping it at the module level

Reply:
same for other two cases i saw this